src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -1,85 +1,175 @@
 package nextstep.subway.line;
 
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
 import nextstep.subway.AcceptanceTest;
+import nextstep.subway.line.dto.LineRequest;
+import nextstep.subway.line.dto.LineResponse;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
 
 @DisplayName("지하철 노선 관련 기능")
 public class LineAcceptanceTest extends AcceptanceTest {
+    LineRequest request;
+
+    @BeforeEach
+    void setRequest() {
+        request = new LineRequest("2호선", "green");
+    }
+
     @DisplayName("지하철 노선을 생성한다.")
     @Test
     void createLine() {
         // when
-        // 지하철_노선_생성_요청
+        ExtractableResponse<Response> response = 지하철노선_생성_요청(request);
 
         // then
-        // 지하철_노선_생성됨
+        지하철노선_응답됨(response, HttpStatus.CREATED);
+    }
+
+    private void 지하철노선_생성됨(ExtractableResponse<Response> response, HttpStatus created) {
+        Assertions.assertThat(response.statusCode()).isEqualTo(created.value());
+    }
+
+    private ExtractableResponse<Response> 지하철노선_생성_요청(LineRequest request) {
+        ExtractableResponse<Response> response = RestAssured
+                .given().log().all()
+                .body(request)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when().post("/lines")
+                .then().log().all().extract();
+        return response;
```

Comment:
```suggestion
        return RestAssured
                .given().log().all()
                .body(request)
                .contentType(MediaType.APPLICATION_JSON_VALUE)
                .when().post("/lines")
                .then().log().all().extract();
```

이렇게 표현한 부분이 많은데 바로 return 해도 될 것 같아요 :)

Reply:
반영하도록 하겠습니다.